xtask/src/xtask.rs

```diff
@@ -246,6 +247,41 @@ fn package(sh: &Shell) -> Result<()> {
     Ok(())
 }
 
+fn update_spec(sh: &Shell) -> Result<Utf8PathBuf> {
```

Comment:
Hmm...but this overlaps with `package-srpm` right? I think in theory we could share code here...or even extract the spec from the srpm?

Reply:
This also relates to https://github.com/containers/bootc/pull/651 and the copr makefile...we've grown way too many copies of this logic.